src/components/views/donate/RecurringDonationCard.tsx

```diff
@@ -76,6 +78,22 @@ export const RecurringDonationCard = () => {
 		// cacheTime: 5_000,
 	});
 
+	// two helper mapValue and mapValueInverse functions generated by copilot - makes the slider range more user friendly
+	function mapValue(value: number) {
+		if (value <= 90) {
+			return value * (50 / 90);
+		} else {
+			return 50 + (value - 90) * (50 / 10);
+		}
+	}
+
+	function mapValueInverse(value: number) {
+		if (value <= 50) {
+			return value * (90 / 50);
+		} else {
+			return 90 + (value - 50) * (10 / 50);
+		}
+	}
```

Comment:
@divine-comedian Thanks, 2 suggestions:
1. please add more comments to describe its functionality
2. please move them to the outside of the component.